### ISSUE TITLE
how to build static "standalone"

### ISSUE THREAD

**Issue by jaronimoe** [uber/kepler.gl#354]
hi,
for me as a novice to webpack, running the demo locally via the commands in the documentation works without a problem.
however, it is not quite clear from the documentation, how I can build my own version of kepler.gl so that I can host it statically on my own server.

apparently upon starting the demo-app via `npm start`, everything is compiled into a bundle.js that is loaded by the index.html.
however, the bundle.js seems to be created at runtime and is not stored in the project folder.

basically, what I want to find out is:
how can I build the project so that I end up with a folder that I can simply push to my server?
for instance, containing an index.html and a bundle.js that work without being launched via node?

**Comment by jaronimoe** [comment]
hi, just wanted to notify you that my issue has been solved with https://github.com/uber/kepler.gl/pull/349

in the version before that, no output library was created and I could not figure out how to adapt the webpack/nodejs config to do that.
